pages/strategy.md

```diff
@@ -0,0 +1,73 @@
+## Introduction
+
+Exercism is a large complex project with thousands of contributors working across dozens of projects, all trying to achieve ambitious goals. It would quickly become overwhelming if we were all working towards making Exercism perfect, so to keep things manageable, we periodically determine the critical next step for Exercism to achieve its goals and then plan that into a strategy.  We present this using the OST framework - defining an **Objective**, which will aim to achieve with a **Strategy**, implemented by a series of **Tactics**.
+
+Our _current_ objective is to make Exercism scalable. 
+
+We expect our next three objectives to be:
+- to make Exercism financially sustainable
+- to make Exercism more accessible to beginners
+- to intentionally grow Exercism for the first time
+
+## Objective
+
+**Make Exercism scalable**
+
+At the time of writing, Exercism has 8,000 new submissions per week. Our mentors currently cannot keep up with the amount of submissions submitted. When they do keep up, the number of submissions almost immediately doubles, as students stick with a Language Track. We are growing by 300-400 new users per day. Many find Exercism too hard to use, but in the future, when that is fixed, we will expect to see the number of submissions double again. We have never actively marketed Exercism. All 300,000 members have come through word of mouth. When we start to market Exercism, we expect to see an increase in usage by two orders of magnitude. 
+
+Our most important goal at this stage is being able to give our current students a timely mentoring experience, and be able to cope with further growth without overwhelming mentors.
+
+We aim to achieve this goal by the end of 2019.
+
+## Strategy
+
+**Acquire and retain more mentors by making mentoring rewarding and enjoyable**
```

Comment:
```suggestion
###Acquire and retain more mentors by making mentoring rewarding and enjoyable
```
If I have missed the point for the change in formatting mid-stream then disregard.